README.md

```diff
@@ -27,7 +27,7 @@ The formats of the dumped assets are either as close as possible to the original
 
 The **Linker** tool allows building custom fastfiles (.ff) for the supported games.
 It can also load existing zones and reuse their assets from in-memory.
-Only a subset of assets can be read from disk however, depending on the game.
```

Comment:
That should probably be "a subset of asset types". If you're already at correcting the README 😅